src/pages/registration/SupportModalContent.tsx

```diff
@@ -1,32 +1,121 @@
-import { useQuery } from '@tanstack/react-query';
-import { useTranslation } from 'react-i18next';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { Trans, useTranslation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
-import { fetchRegistrationTickets } from '../../api/registrationApi';
+import { addTicketMessage, createTicket, fetchRegistrationTickets } from '../../api/registrationApi';
 import { setNotification } from '../../redux/notificationSlice';
 import { Registration } from '../../types/registration.types';
-import { ActionPanelContent } from '../public_registration/ActionPanelContent';
+import { Box, Divider, Grid, Link as MuiLink, Skeleton, Typography } from '@mui/material';
+import { MessageForm } from '../../components/MessageForm';
+import { OpenInNew } from '@mui/icons-material';
 
 interface SupportModalContentProps {
   closeModal: () => void;
   registration: Registration;
 }
 
-export const SupportModalContent = ({ registration }: SupportModalContentProps) => {
+export const SupportModalContent = ({ closeModal, registration }: SupportModalContentProps) => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
 
   const ticketsQuery = useQuery({
     enabled: !!registration,
     queryKey: ['registrationTickets', registration.id],
-    queryFn: () => fetchRegistrationTickets(registration.id),
+    queryFn: async () => await fetchRegistrationTickets(registration.id),
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_tickets'), variant: 'error' })),
   });
 
+  const addMessageMutation = useMutation(async (data: any) => await addTicketMessage(data.ticketId, data.message), {
```

Comment:
Kanskje heller definere en egen type for denne?
`interface AddMessageData {
  ticketId: string;
  message: string;
}`
```suggestion
  const addMessageMutation = useMutation(async (data: AddMessageData) => await addTicketMessage(data.ticketId, data.message), {
```

Reply:
Fikser